StreamingKit/StreamingKit/STKQueueEntry.m

```diff
@@ -65,6 +65,12 @@ -(double) duration
         return 0;
     }
     
+    if (audioPacketCount > 0.0)
```

Comment:
You'll definitely want an additional check here that it's a CBR format, not sure if the final packet would be partial as well?

Reply:
It should be able to handle VBR as it uses frame duration in packets and they should be constant.